egs/multi_en/s5/conf/ami_ihm/mfcc.conf

```diff
@@ -0,0 +1,4 @@
+--use-energy=false
+--sample-frequency=16000
```

Comment:
@danpovey: So you'd recommend switching to downsampling and retraining the HMM-GMM models from scratch before starting TDNN training?